automated-testing/cypress.env.json.remote-dev

```diff
@@ -15,5 +15,5 @@
     {"token": "SAS", "name": "SAS"},
     {"token": "Matlab", "name": "Matlab"}
   ],
-  "support_ticket": {"creationEnabled": false}
+  "support_ticket": {"creationEnabled": true, "queue": "Internal:Logging"}
```

Comment:
creation enabled in `remote-dev` will require the production `rt_config` file to be added manually to the dashboard folder before deploying.

Reply:
This is documented in detail [here](https://wiki.harvard.edu/confluence/display/HMDC/Deploying+Sid2+to+Remote+Dev), which is referred to [here](https://github.com/hmdc/sid2/blob/731-multiple-support-queues/dashboard/README.md#rt-configuration-for-remote-dev).